test/e2e/lib/driver-helper.js

```diff
@@ -24,45 +24,63 @@ export async function highlightElement( driver, element ) {
 	}
 }
 
-export function clickWhenClickable(
-	driver,
-	selector,
-	waitOverride = null,
-	extraErrorString = null
-) {
-	const timeoutWait = waitOverride ? waitOverride : explicitWaitMS;
-	const extraErrorStringAppend = extraErrorString ? ' ' + extraErrorString : '';
+const until = {
+	...webdriver.until,
+	/**
+	 * Creates a condition that will loop until element is clickable. A clickable
+	 * element must be located and not (aria-)disabled.
+	 *
+	 * @param {By} locator The element's locator
+	 * @returns {WebElementCondition} The new condition
+	 */
+	elementIsClickable( locator ) {
+		const locatorStr = typeof locator === 'function' ? 'by function()' : locator + '';
+
+		return new WebElementCondition(
+			`for element to be clickable ${ locatorStr }`,
+			async function ( driver ) {
+				try {
+					const element = await driver.findElement( locator );
+					const isEnabled = await element.isEnabled();
+					const isAriaEnabled = await element
+						.getAttribute( 'aria-disabled' )
+						.then( ( v ) => v !== 'true' );
+
+					return isEnabled && isAriaEnabled ? element : null;
```

Comment:
What if / could it happen that the element `isEnabled` but not `isAriaEnabled`? The `&&` here seems to suggest that it might be possible.

Reply:
Good question, because it's how we actually disable elements in Gutenberg (most probably because of the [consistency issue between screen readers](https://a11y-101.com/development/aria-disabled)). So when an element is enabled, but at the same time it's `aria-disabled="true"` it should be treated just like a disabled element would, which in the result should be a non-clickable/non-actionable element.